app/code/community/OnePica/AvaTax/Model/Records/Queue/Process.php

```diff
@@ -158,7 +158,7 @@ protected function _parseInvoices()
             try {
                 $invoice = Mage::getModel('sales/order_invoice')->load($item->getEntityId());
                 if ($invoice->getId()) {
-                    Mage::getModel('avatax/avatax_invoice')->invoice($invoice, $item);
+                    Mage::getModel('avatax/calculator')->invoice($invoice, $item);
```

Comment:
Move model init to separate method and check if it possible to use getSingleton instead of getModel